sympy/stats/joint_rv.py

```diff
@@ -161,6 +180,9 @@ class JointRandomSymbol(RandomSymbol):
     def __getitem__(self, key):
         from sympy.stats.joint_rv import JointPSpace
         if isinstance(self.pspace, JointPSpace):
+            if Le(self.pspace.component_count, key):
```

Comment:
this is not guaranteed to result in a boolean, I believe. I would suggest `(val1 <= val2) == True`